node1: "obstacle_2d"
topic1: "velodyne_points"
rate1: 1.0
topic2: "scan"
rate2: 40.0
topic3: "ndt_pose"
rate3: 1.0
num_topics: 3
num_nodes: 1
min_rate: 0.5